docs/src/layouts/links.js

```diff
@@ -80,6 +80,7 @@ const ComponentsLinks = [
         route: routes.CHECKBOX,
         children: 'Checkbox',
         tags: ['android', 'web'],
+        keywords: ['switch', 'toggle'],
```

Comment:
Example of adding keywords.

![](http://1.bp.blogspot.com/_GVhQClYk7TM/SGc_NFJjvTI/AAAAAAAAB_w/Q3G-6F2aIs8/s320/Art_Attack.gif) Try typing 'toggle' and notice that checkbox is in the results 🙌